src/controller/python/test/test_scripts/base.py

```diff
@@ -386,6 +386,37 @@ def TestFailsafe(self, nodeid: int):
             return True
         return False
 
+    async def TestControllerCATValues(self, nodeid: int):
+        ''' This tests controllers using CAT Values
+        '''
+
+        # Allocate a new controller instance with a CAT tag.
+        newController = self.fabricAdmin.NewController(nodeId=300, catTags=[0xFFFFFFFD00010001])
```

Comment:
The CAT value is `uint32_t` (i.e. `0x00010001`) - will this 64-bit value be truncated to the 32-bit value? 

We have a helper function `CASEAuthTagFromNodeId()` for that

Reply:
Good catch. It will get truncated, but it should actually just be the 32-bit numbers. Will fix.